src/test/java/com/portatlas/DirectoryTest.java

```diff
@@ -0,0 +1,55 @@
+package com.portatlas;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+public class DirectoryTest {
+
+    private Directory dir;
+
+    @Before
+    public void setUp() {
+        dir = new Directory();
+    }
+
+    @Test
+    public void testGetDefaultDir() {
+        assertEquals(System.getProperty("user.dir") + "/public/", dir.defaultDir);
+    }
+
+    @Test
+    public void testPrintFileInDir() {
+        final File folder = new File(dir.defaultDir);
+
+        List<String> files = Arrays.asList("file1",
+                                           "file2",
+                                           "image.gif",
+                                           "image.jpeg",
+                                           "image.png",
+                                           "partial_content.txt",
+                                           "patch-content.txt",
+                                           "text-file.txt");
+
+        assertEquals(files , dir.listFilesForFolder(folder));
```

Comment:
Yes, the test depends on the files that is in the /public directory. I could see how this would be an issue if I was serving new resources or if the directory no longer exists. I guess another way to test is just to check if it has any content but I wanted to check to make sure I was actually showing the contents in that directory.